0.data_analysis/plate_5_characteristics/nbconverted/plate_5_sc_feature_selected_camerons_agg_well_corr.py

```diff
@@ -96,14 +98,41 @@
 # In[7]:
 
 
-cp = CorrelatePlate()
+cd = CorrelateData()
+correlationsdf = []
 
-# Correlates aggregated wells both between and across groups
-correlationsdf = cp.correlate_agg_wells(welldf, "Metadata_Well", feat_cols, "Metadata_genotype")
+# Correlates aggregated wells across genotype
+correlationsdf.append(cd.inter_correlations(
```

Comment:
This is a stylistic suggestion, but I find code more readable when you include the parameter name in the function (e.g., `_df = welldf.reset_index(drop=True)`, etc.).

Reply:
Changed this to show the parameter names